### PR TITLE
yetus: disable hadolint DL3059 and shellcheck SC3043

### PR DESCRIPTION
**Pull request by christoph-zededa**
this check informs that a consolidation of "RUN"s in Dockerfiles should be considered, but even the developers of hadolint themselves admit that there are drawbacks: https://github.com/hadolint/hadolint/wiki/DL3059

Besides that it disables the use of the cache of docker it IMO makes the developer write less-readable code with long concatenations of shell commands for one "RUN".

### CODE REVIEW

File: pkg/bsp-imx/Dockerfile [hunk 1]
```diff
@@ -80,7 +80,6 @@ ENV UBOOT_FILES_imx8mp_pollux "flash.bin"
 ENV UBOOT_CONFIG_imx8mp_pollux "phycore-imx8mp"
 ENV UBOOT_DTB_imx8mp_pollux "arch/arm/dts/imx8mp-phyboard-pollux-rdk.dtb"
 ENV FLASH_OFFSET_imx8mp_pollux "32"
-# hadolint ignore=DL3059,SC2086
 RUN for target in ${UBOOT_TARGETS}; do \
```
Comment (eriknordmark) [inline pkg/bsp-imx/Dockerfile:83]: Looks like yetus/hadlint still complains about SC2086 here.

File: pkg/u-boot/Dockerfile [hunk 2]
```diff
@@ -53,9 +53,7 @@ FROM build-${TARGETARCH} AS build
 COPY config /tmp/
 RUN cat /tmp/config >> "configs/${TARGET}"
 # generate default config for the target
-# hadolint ignore=DL3059,SC2086
 RUN make $TARGET
```
Comment (eriknordmark) [inline pkg/u-boot/Dockerfile:56]: Ditto

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Still need the hadlint ignore for SC2086 in two places

**Comment by rene** [comment]
@christoph-zededa , nice PR, I really liked the approach with .hadolint.yaml file, it increases a lot the readability of the code.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**